tests/core/test_plan.py

```diff
@@ -189,6 +191,55 @@ def test_restate_models(sushi_context_pre_scheduling: Context):
         sushi_context_pre_scheduling.plan(restate_models=['"unknown_model"'], no_prompts=True)
 
 
+@freeze_time()
```

Comment:
Would it be better to freeze at a specific time? That way you don't have to worry about day switching issues. Also you could hardcode values in the test.

Reply:
I'm freezing here to prevent day switching issue. The specific day doesn't really matter and the current timestamp at the time of freezing works just fine.

Reply:
Ok nevermind not an issue.